src/sentry/models/group.py

```diff
@@ -718,5 +736,26 @@ def issue_category(self):
 @receiver(pre_save, sender=Group, dispatch_uid="pre_save_group_default_substatus", weak=False)
 def pre_save_group_default_substatus(instance, sender, *args, **kwargs):
     if instance:
-        if instance.status == GroupStatus.UNRESOLVED and instance.substatus is None:
-            instance.substatus = GroupSubStatus.ONGOING
+        # We only support substatuses for UNRESOLVED and IGNORED groups
+        if (
+            instance.status not in [GroupStatus.UNRESOLVED, GroupStatus.IGNORED]
+            and instance.substatus is not None
+        ):
+            raise ValueError(
```

Comment:
I _wonder_ if we should use `logger.exception` first and then on a follow up PR make all the ValueError changes.

What do you think?

Maybe I'm paranoid.

Reply:
that's a great idea - we can let it run for a week and switch to ValueError if things look safe